### DISCUSSION TITLE
Cosmetic - Text background color

### DISCUSSION THREAD

**Discussion by cuda444** [djdd87/SynoAI#124] [Q&A] [answered]
Forgive me if this is a newb question, but is there a way to add a background color to the text in the detection box (object and % certainty)? The text itself is difficult to read on some backgrounds due to the contrast. Was thinking a green text box, for example, with transparent, black or white text would be more visible.

Is this doable?

Thanks for all your efforts on this. It has made the overall experience SO much better.

**Comment by djdd87** [comment, marked as answer]
Yes that's doable. Raise an issue with your requirements and I'll get to it one day.

**Comment by cuda444** [comment]
Thanks! And thanks for the quick reply.
